src/slice.cr

```diff
@@ -239,6 +239,68 @@ struct Slice(T)
     pointer(count).copy_to(target, count)
   end
 
+  # Copies the contents of this slice into *target*.
+  #
+  # Truncates if this slice doesn't fit into the destination slice.
```

Comment:
I don't think silently truncating is good. If there's not enough room it should raise.

Are you using this "feature" in your code?